decentraland/unity-renderer
# Feat: adding changes to profile card in explore menu

## What does this PR change?

Fix #3329 
Adds changes to the profile card in explore menu for the new passport flow

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/update-profile-card
2. Open the explore menu v2
3. Verify the new user card in top right
4. Check everything that is cited in issue #3329 works as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
